.drone.yml

```diff
@@ -5,4 +5,4 @@ build:
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
     - pip install python-coveralls
-    - make run-tests
+    - lettuce ensime_shared/spec/features
```

Comment:
I think, from reading without actually trying to change it yet, that it would be idempotent and not wastefully run `pip install` when not needed, as long as `$(ACTIVATE)` exists and `requirements.txt` hasn't changed, since it includes the `test -d .venv` check. IMO it could be refactored a little bit to be more clear and let make do the work, maybe sort of like this (untested):

```make
# Moved to first task to be the implicit default
run-tests: requirements
	# ...

$(ACTIVATE):
	virtualenv -p python2 .venv

requirements: $(ACTIVATE) requirements.txt
	. $(ACTIVATE); $(REQS)
	. $(ACTIVATE); $(TESTREQS)

lint: requirements
	# ...

autopep8: requirements
	# ...
```

Also some/most of the targets technically should be declared as `.PHONY`. The newly-working-again test suite was just merged, I've never really looked at the `Makefile` until now 😇